docs/website/docs/installation.mdx

```diff
@@ -8,4 +8,102 @@ import Tabs from '@theme/Tabs';
 
 ## Set up environment
 
-## Install `dlt` library
\ No newline at end of file
+Make sure you are using **Python 3.8 or 3.9**:
+
+```bash
+python3 --version
+pip3 --version
+```
+
+If not, then please follow the instructions below to install it:
+
+<Tabs values={[{"label": "Ubuntu", "value": "ubuntu"}, {"label": "macOS", "value": "macos"}, {"label": "Windows", "value": "windows"}]}  groupId="operating-systems" defaultValue="ubuntu">
+  <TabItem value="ubuntu">
+
+You can install Python 3.9 with an `apt` command.
+
+```bash
+sudo apt update
+sudo apt install python3.9
+```
+
+  </TabItem>
+  <TabItem value="macos">
+
+Once you have installed [Homebrew](https://brew.sh), you can install Python 3.9.
+
+```bash
+brew update
+brew install python@3.9
+```
+
+  </TabItem>
+  <TabItem value="windows">
+
+Once you have installed the [Microsoft VC++ Compiler](https://visualstudio.microsoft.com/visual-cpp-build-tools/), you can install [Python 3.9](https://www.python.org/downloads/windows/) (64-bit version) for Windows.
```

Comment:
I must verify this. we should have no dependencies that require compilation. where did you take this info from?

Reply:
Sweet. If don't need this, even better!